imgui-examples/examples/hello_world.rs

```diff
@@ -4,26 +4,32 @@ mod support;
 
 fn main() {
     let system = support::init(file!());
+
     let mut value = 0;
     let choices = ["test test this is 1", "test test this is 2"];
+    let mut open = true;
+
     system.main_loop(move |_, ui| {
-        ui.window("Hello world")
-            .size([300.0, 110.0], Condition::FirstUseEver)
-            .build(|| {
-                ui.text_wrapped("Hello world!");
-                ui.text_wrapped("こんにちは世界！");
-                if ui.button(choices[value]) {
-                    value += 1;
-                    value %= 2;
-                }
+        if open {
```

Comment:
Minor thing but should possibly put this in a different example as would be good to keep `hello_world` example simple

Reply:
True! Should probably be reverted